Added example descriptions to ns-datepicker (#654)
centrica-engineering/nucleus-docs: src/content/docs/components/ns-datepicker.mdx

    <ExampleViewer type="standard" />
  </TabItem>
  <TabItem label="Standard min max">

>   The `min` and `max` attributes can be used to define a selectable range between two specific dates, or each can be used separately as a limit.

Reviewer: What is the purpose of setting a min or max limit? Describing a common scenario may help here. Link to the Spec if relevant.

    <ExampleViewer type="standard-min-max" />
  </TabItem>
  <TabItem label="Mask">

>   Use `type="mask"` when the calendar feature is not required.

Reviewer: I kinda think that the purpose should be positive. 

So; why is this type required? Rather than why another type is not required.

>   A `mask` type can be used for inputting a specific date in a set sequence, providing a more efficient alternative to scrolling through a calendar, especially when entering dates such as a date of birth.

Reviewer: ```suggestion
  A `mask` type can be used for inputting a specific date, providing an efficient alternative to scrolling through a calendar, especially when entering dates such as a date of birth.
```

    <ExampleViewer type="mask" />
  </TabItem>
  <TabItem label="Mask min max">

>   The `min` and `max` attributes can be used to define a selectable range between two specific dates, or each can be used separately as a limit.

Reviewer: What is the purpose of setting a min or max limit? Describing a common scenario may help here. Link to the Spec if relevant.
